### ISSUE TITLE
loss nan when training fine-tuning based methods

### ISSUE THREAD

**Issue by Fly-dream12** [open-mmlab/mmfewshot#18]
Thanks for your code.
When i try the fine-tuning based methods such as TFA and FSCE, the loss become nan at about 250 iters. I tried by decreasing the lr, but it doesn't work. Could you give some suggestions?

**Comment by linyq17** [comment]
Please refer to `https://github.com/open-mmlab/mmdetection/blob/master/docs/faq.md#training`.
